ingestors/kafka/src/main/scala/hydra/kafka/model/TopicMetadataV2Transport.scala

```diff
@@ -52,9 +53,16 @@ object ContactMethod {
   }
 }
 
+object Schemas {
+  def emptySchema: Schema = SchemaBuilder
```

Comment:
When would this function actually be used? Is this just in sending data back to the front-end if the schema doesn't exist?

Reply:
That is exactly right.

Reply:
Would it make more sense to make the field optional in the metadata when we send it down to the front end?

Reply:
Just explodes issues throughout the project. We felt like this solution minimizes that. The change can be made, we just need to decide what to do with optional schemas throughout the project where they really can't be optional (it's a parse error if they don't provide it on the way in).

Reply:
Makes sense. I think that I worry about what the `emptySchema` will cause to happen throughout the project. Could be a crutch that allows us to ignore error cases that are important.

If the schema has to be there, and truly isn't optional, then we should have it show up like the error that it is. If we read in a piece of metadata that does not have an associated schema, we should probably just not add that metadata to the cache and log out a `WARN`. If we wanted to, we could even keep a separate list of metadata or topic names that have no schema.. idk Thoughts?